pages/common/conda.md

```diff
@@ -0,0 +1,35 @@
+# conda
+
+> Package, dependency and environment management for any programming language.
+
+- Create a new environment, installing named packages:
```

Comment:
Not sure if it would be nice to give an example of common packages? _e.g.:_ `python=3.4`